### ISSUE TITLE
About the image size of Tensorflow

### ISSUE THREAD

**Issue by ideaRunner** [chengjunwen/spp_net#2]
Hi, chengjunwen:

I think in tensorflow, the convolutional layer's input image size must be fixed.
How you make it be capable to input different image size ?

Your reply will be really appriciated, thank you!

**Comment by danielkaifeng** [comment]
I agree. I think in tf.placeholder(), you must feed a fixed shape so that in later spp layer you can calculate the previous conv layer shape.

**Comment by danielkaifeng** [comment]
In `spp_layer.py`, if the size of  not fixed previous conv layer will be `None`, so error occur when you try to devide it.
`self.batch_size = self.input.get_shape().as_list()[0]`

**Comment by WanliOuyang** [comment]
@danielkaifeng  How to solve this problem?

**Comment by danielkaifeng** [comment]
@WanliOuyang Unfortunately there is no way to solve this in tensorflow to my knowledge.

**Comment by WanliOuyang** [comment]
@danielkaifeng When I use a fixed batch size value(such as batch_size=4), the code can run, but when training, the precision of the training keeps fluctuating on a very small number(such as the mean accuracy is 0.16). Why is this?

**Comment by world2025** [comment]
@danielkaifeng Do you sovle it and do you know how to use the multi_scale image inputs ?

**Comment by world2025** [comment]
@ideaRunner  Hi, do you solve the multi-scale image input size problem ?
